### ISSUE TITLE
Pick a consistent code style

### ISSUE THREAD

**Issue by Joshua-Ashton** [Plagman/gamescope#347]
I always never have any idea what style to go for when programming for gamescope. I usually just go with whatever is around me at the time.

We should probably just pick some clang_format given the state of things, fix the variable names, and go for that at this point.

**Comment by Plagman** [comment]
I don't think we should try to fix the code style just for the sake of it on existing code, but picking something and making sure new code, and code that gets touched, uses it, is fine.
